www/messenger.js

```diff
@@ -0,0 +1,60 @@
+//import Chart from 'chart.js/auto'
```

Comment:
This code should also go inside of Brady's. We shouldn't have any js or html in our backend code base.

Reply:
same comment as above